docs/Operators.md

```diff
@@ -18071,7 +18071,7 @@ Other versions of this operator: <a href="Changelog.md#PRelu-1">1</a>, <a href="
 <dt><tt>X</tt> (differentiable) : T</dt>
 <dd>Input tensor</dd>
 <dt><tt>slope</tt> (differentiable) : T</dt>
-<dd>Slope tensor. The shape of slope can be smaller then first input X; if so, its shape must be unidirectional broadcastable to X</dd>
+<dd>Slope tensor. The shape of slope can be smaller than first input X; if so, its shape must be unidirectional broadcastable to X</dd>
```

Comment:
Thanks for doing this. This file is automatically generated so you need to modify this file as well: https://github.com/onnx/onnx/blob/main/onnx/defs/math/defs.cc#L710.